app/views/works/_required_form.html.erb

```diff
@@ -2,14 +2,14 @@
   <div>
     <p class="doi-text">By initiating this new submission, we have reserved a draft DOI for your use. This unique
       identifier will become searchable at the completion of this process.</p>
-    <input class="text-not-editable" type="text" id="doi" name="doi" value="<%= @work.doi %>" readonly/>
+    <input class="text-not-editable" type="text" id="doi" name="doi" value="<%= @work.resource.doi %>" readonly/>
   </div>
   <div>&nbsp;</div>
 <% else %>
   <div class="field">
     <p class="doi-text">By initiating this new submission, we have reserved a draft DOI for your use.<br/>
       This unique identifier will become fully minted and searchable at the completion of this process.</p>
-    <p>DOI: <%= @work.doi %></p>
+    <p>DOI: <%= @work.resource.doi %></p>
```

Comment:
I wonder with all the places that we are reaching into the resource for doi if we should have a helper method like title, so we can still call work.doi.

Reply:
I'll ticket this as a separate issue. For now I want to have a way to still fetch the `work.doi` from the ActiveRecord so that I can migrate it to `work.resource.doi`

Reply:
https://github.com/pulibrary/pdc_describe/issues/319